modular-player/webpack/src/index.js

```diff
@@ -0,0 +1,42 @@
+import { Player } from 'bitmovin-player/modules/bitmovinplayer-core';
+import EngineBitmovinModule from 'bitmovin-player/modules/bitmovinplayer-engine-bitmovin';
+import MseRendererModule from 'bitmovin-player/modules/bitmovinplayer-mserenderer';
+import HlsModule from 'bitmovin-player/modules/bitmovinplayer-hls';
+import AbrModule from 'bitmovin-player/modules/bitmovinplayer-abr';
+import ContainerTSModule from 'bitmovin-player/modules/bitmovinplayer-container-ts';
+import SubtitlesModule from 'bitmovin-player/modules/bitmovinplayer-subtitles';
+import SubtitlesCEA608Module from 'bitmovin-player/modules/bitmovinplayer-subtitles-cea608';
+import PolyfillModule from 'bitmovin-player/modules/bitmovinplayer-polyfill';
+import StyleModule from 'bitmovin-player/modules/bitmovinplayer-style';
+
+import { UIFactory } from 'bitmovin-player/bitmovinplayer-ui';
+import 'bitmovin-player/bitmovinplayer-ui.css';
```

Comment:
Since you don't use the UI package any more you can remove it from the package.json